### PR TITLE
update cluster detach senario

### PR DESCRIPTION
**Pull request by leena-jawale**
Signed-off-by: Leena Jawale <ljawale@redhat.com>

### CODE REVIEW

File: quick_start.adoc [hunk 1]
```diff
@@ -48,6 +48,8 @@ Now a placement can be created.  This will be used to determine which managed cl
 [IMPORTANT]
 ====
 The identity configuration management for Kubernetes Technology Preview does not yet utilize custom role definitions for controlling access to work with AuthRealms. Instead, it relies on the role-based access control implemented by multicluster engine and Advanced Cluster Management around ManagedClusterSets, ManagedClusterSetBindings, and Placements. AuthRealms must exist in a namespace containing a Placement, and the Placement leverages the ManagedClusterSets that are bound to that namespace. Access to namespaces with existing bound ManagedClusterSets and/or existing Placements should be managed accordingly.
+
+Before detaching cluster from hub, copy original OAuth from the ConfigMap named `idp-oauth-original` in the managed cluster's namespace on the hub cluster. After cluster is completely detached from hub cluster, manually edit the OAuth on the cluster which was managed by hub and update only spec from copied ConfigMap to OAuth. You should restore original OAuth on the managedcluster.
```
Comment (robinbobbitt) [inline quick_start.adoc:52]: I think we need to handle this with two PRs.. One for now before your change that orphans the IDP Oauth config goes live, and another one that we hold until then with the info you provide above (or tell them to take it out of management before detaching it, and if they didn't, then follow your guidance). For now, we need to add an `IMPORTANT` note that they need to make sure the cluster is no longer under IDP config management before detaching it (by making sure it doesn't match the placement... remove labels or remove from managed cluster set) or they will lose access to the cluster. I think this should probably be up in the section about adding clusters to the managed cluster set, or at least be separated from this RBAC note into its own `IMPORTANT` section.

**Review by robinbobbitt — CHANGES_REQUESTED** [review]

**Comment by leena-jawale** [comment]
/hold
Until release

File: quick_start.adoc (hunk 1 above)
Comment (robinbobbitt) [reply quick_start.adoc:54]: @leena-jawale please see https://github.com/identitatem/idp-mgmt-docs-upstream/pull/2

**Comment by robinbobbitt** [comment]
Closing as we are now making changes to the upstream repo. Recommended changes are here: https://github.com/identitatem/idp-mgmt-docs-upstream/pull/2

**Pull request closed without merging**